test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -30,6 +31,13 @@
 def test_ec2_tensorflow_inference_neuron(tensorflow_inference_neuron, ec2_connection, region):
     run_ec2_tensorflow_inference(tensorflow_inference_neuron, ec2_connection, "8500", region)
 
+@pytest.mark.model("mnist")
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_NEURONX_ACCELERATOR_TYPE, indirect=True)
+# FIX ME: Sharing the AMI from neuron account to DLC account; use public DLAMI with inf1 support instead
+@pytest.mark.parametrize("ec2_instance_ami", [test_utils.UL20_PT_NEURON_US_WEST_2], indirect=True)
```

Comment:
Lets fix the ami id if not already

Reply:
It is fixed to a DLAMI that provides the correct neuron driver version (which is all we need), not to an exact ID (same as the others in that test, i didn't change anything)